homeassistant/components/homeassistant_hardware/silabs_multiprotocol_addon.py

```diff
@@ -356,12 +356,12 @@ async def async_step_show_revert_guide(
         """Link to a guide for reverting to Zigbee firmware."""
         if user_input is None:
             return self.async_show_form(step_id="show_revert_guide")
-        return self.async_create_entry(title="", data={})
+        return self.async_create_entry(data={})
```

Comment:
Shouldn't the steps that basically aborts the flow, be replaced with `async_abort` instead and an abort reason.